feat: Initial commit of CIS module (#349)
terraform-ibm-modules/terraform-ibm-cis: modules/glb/README.md

> # CIS Global Load Balancer (GLB) Module

Reviewer: ```suggestion
# CIS global load balancer module
```

Author: done

> This module is used to provision Global Load Balancer that include load balancers, origin pools and health checks.

Reviewer: lowercase global load balancer here as everywhere else

Author: made the required changes

> **Note:**

Reviewer: remove the note label

> When `glb_proxied` is set as `true`, then `ttl` is automatically set and cannot be updated. Hence, when `terraform plan` is run after a successful `terraform apply`, you will receive the following message but your infrastructure will remain intact.

Reviewer: Sugges that you update like in the dns module readme

# CIS global load balancer (GLB) module

> This module is used to provision global load balancer that include load balancers, origin pools and health checks.

Reviewer: ```suggestion
This module provisions a global load balancer that includes load balancers, origin pools, and health checks.
```

Author: Made the required changes.

When `glb_proxied` is set as `true`, then `ttl` is automatically set and cannot be updated.

> This means that when you run a `terraform plan` command after a successful `terraform apply`, the output shows that the ttl value requires an update, as shown in the following example. However, your infrastructure will not be affected.

Reviewer: Nitpick: I suggest either using the abbreviation (TTL) or putting the variable name in backticks (`ttl`). Might have missed this earlier.

Author: sure, made the changes as `ttl`
